cmd/git-sync/main.go

```diff
@@ -183,6 +189,36 @@ func envDuration(key string, def time.Duration) time.Duration {
 	return def
 }
 
+func sanitizeSpecificTime(t string) (h int, m int) {
+	if uniseg.GraphemeClusterCount(t) != 4 {
+		err := errors.New("error=")
+		log.Error(err, "--specific-time value not in military format, e.g. 1500 or 0130")
+		os.Exit(1)
+	}
+
+	tsplit := strings.Split(t, "")
+	h, _ = strconv.Atoi(strings.Join(tsplit[0:2], ""))
+	m, _ = strconv.Atoi(strings.Join(tsplit[2:], ""))
+
+	return h, m
+}
+
+func initSpecific(h int, m int) {
+	hold := true
+	t := time.Now()
+	n := time.Date(t.Year(), t.Month(), t.Day(), h, m, 0, 0, t.Location())
+	d := n.Sub(t)
+	if d < 0 {
+		n = n.Add(24 * time.Hour)
+		d = n.Sub(t)
+	}
+	for hold {
+		time.Sleep(d)
+		d = 24 * time.Hour
+		hold = false
+	}
+}
```

Comment:
Comment this method. I dont understand whats going on here.

you get the current time, then subtract the given? and this gives you what time to wait?